src/main.py

```diff
@@ -439,8 +439,8 @@ def metric_page():
     )
 
     fig6.update_layout(yaxis_range=[0, MAE_LIMIT_DEFAULT])
-    st.plotly_chart(fig6, theme="streamlit")\
-
+    st.plotly_chart(fig6, theme="streamlit")
+    st.text("PV live if the different between the intraday and day after PV Live values.")
```

Comment:
useful information. couple typos in here that I'll clean up.

Reply:
I'm happy to fix them